packages/typescript-estree/src/parseSettings/index.ts

```diff
@@ -123,6 +118,11 @@ export interface MutableParseSettings {
    */
   projects: ReadonlyMap<CanonicalPath, string>;
 
+  /**
+   * Experimental: TypeScript server to power program creation.
+   */
+  projectService: ProjectServiceSettings | undefined;
```

Comment:
```suggestion
  /**
   * TypeScript server to power program creation.
   */
  projectService: ProjectServiceSettings | undefined;
```


Since it's considered stable, I don't think the documentation should say it's experimental. WDYT?